src/pages/login/CertificateGenerate.tsx

```diff
@@ -28,191 +21,49 @@ const CertificateGenerate: FC = () => {
-                      <Col size={6}>
-                        <p>
-                          Download the <code>.crt</code> file and add it to the
-                          LXD trust store
-                        </p>
-                      </Col>
-                      {certs && (
-                        <Col size={2}>
-                          <Button
-                            className="download-crt"
-                            onClick={() => {
-                              downloadText(crtFileName, certs.crt);
-                            }}
-                          >
-                            Download&nbsp;crt
-                          </Button>
-                        </Col>
-                      )}
-                    </Row>
-                    <div className="p-code-snippet">
-                      <pre className="p-code-snippet__block--icon">
-                        <code>
-                          lxc config trust add Downloads/
-                          {crtFileName}
-                        </code>
-                      </pre>
-                    </div>
-                  </div>
-                </Col>
-              </Row>
-            </li>
-            <li className="p-stepped-list__item">
-              <Row>
-                <Col size={3}>
-                  <h2 className="p-stepped-list__title p-heading--5">Import</h2>
-                </Col>
-                <Col size={8}>
-                  <BrowserImport
-                    sendPfx={
-                      certs
-                        ? () => {
-                            downloadBase64(
-                              `lxd-ui-${location.hostname}.pfx`,
-                              certs.pfx,
-                            );
-                          }
-                        : undefined
-                    }
-                  />
-                </Col>
-              </Row>
-            </li>
-            <li className="p-stepped-list__item u-no-margin--bottom">
-              <Row>
-                <Col size={3}>
-                  <h2 className="p-stepped-list__title p-heading--5">Done</h2>
-                </Col>
-                <Col size={6}>
-                  <div className="p-stepped-list__content">
-                    <p>Enjoy LXD UI.</p>
-                  </div>
-                </Col>
-              </Row>
-            </li>
-          </ol>
+              It looks like you you already have a client certificate installed
+              and selected, skip to the next step.
+            </Notification>
+          )}
+          <Notification
+            actions={[
+              ...(hasCertificate
+                ? [
+                    {
+                      label: "I already have a client certificate",
+                      onClick: () => {
+                        navigate("/ui/login/certificate-add");
+                      },
+                    },
+                  ]
+                : []),
+            ]}
```

Comment:
I think the branches here should be inverted. If they have a cert, we should not show the link. Because the other notification already has a link in that case.